package.json

```diff
@@ -38,6 +38,7 @@
     "parse-server-push-adapter": "1.0.4",
     "parse-server-s3-adapter": "1.0.4",
     "parse-server-simple-mailgun-adapter": "1.0.0",
+    "parse-aws-firehose-logger-adapter": "acinader/parse-aws-firehose-logger-adapter",
```

Comment:
i buy that.  fwiw, I added so that i could continue to run parse-server from the command line instead of creating a wrapper to call the constructor.  i'll remove this, the require in index, and the test in AdapterLoaderSpec.  

Once I've tested with the wrapper, I'll push the changes to this pr and write the readme for https://github.com/acinader/parse-aws-firehose-logger-adapter